triplea_maps.yaml

```diff
@@ -434,12 +434,32 @@
     <br>Use the special tech tree system to gain access to specific units.
     <br>
     <br>
+- mapName: Another World
+  mapCategory: GOOD
+  url: https://raw.githubusercontent.com/triplea-maps/another_world/master.zip
+  version: 3.7
```

Comment:
Version should just be 1. This isn't the map version. Just used to increment to indicate to the engine that there is a new download.